File: C#/DTO/Booking.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DTO
{
    public class Booking
    {
        public short BookingId { get; set; }

        public short? UserId { get; set; }

        public DateTime? DateBooking { get; set; }

        public TimeSpan? BookingTime { get; set; }

        public short? TripId { get; set; }

        public int? SeveralPlaces { get; set; }
        public string UserName { get; set; }

        public string TripTarget { get; set; }
        public DateTime TripDate { get; set; }

       
    }
}

File: Angular/src/app/class/Booking.ts
import { Time } from "@angular/common";

export class Booking{
    constructor(       
        public  bookingId?:number,
        public  userName?: string,
        public  tripTarget?: string,
        public  tripDate?: Date,
        public  userId?: number,
        public  dateBooking?: Date,
        public  bookingTime?:Time, 
        public  tripId?: number,
        public  severalPlaces?:number)
          {
          }
  }
  



